modules/persistent-storage-csi-migration-vsphere-to-4-13.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-vsphere-to-4.13_{context}"]
+= Updating from {product-title} 4.12 to 4.13 with vSphere in-tree PVs
+
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.13, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} update can occur and automatic CSI migration for vSphere occurs only if you opt in.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by using one of the following methods:
+
+* Initiating the upgrade from a {product-title} 4.12 Z-stream.
+
+* Performing an administrator acknowledgment by running the following command:
++
+[source, terminal]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-1.26-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
+
+[IMPORTANT]
+====
+If you update to {product-title} 4.13 without updating vSphere, and then opt in to migration, known issues can occur. https://access.redhat.com/node/7011683[Before updating, carefully read this knowledge base article].
```

Comment:
add 4.13 link to opt in ?
https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/persistent-storage-csi-migration.html#upgrading-openshift_persistent-storage-csi-migration

Reply:
Done.